runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1758,6 +1767,13 @@ public void reportPeriodicWorkerUpdates() {
     } catch (Exception e) {
       LOG.error("Unexpected exception while trying to send counter updates", e);
     }
+    try {
+      sendWorkerMessage();
```

Comment:
do we want to do this for appliance or just streaming engine?

Reply:
maybe do this from a separate periodic method than the existing method sending the counters?
So that the latency of reporting this message doesn't hold up counters which can affect watermark latency in appliance.

Reply:
Just streaming engine for now. Create a new periodic method to send the WorkerMessage